daemon/system-info.sh

```diff
@@ -304,8 +305,7 @@ if [ "${KERNEL_NAME}" = "Darwin" ]; then
         fi
 
         DISK_DETECTION="df"
-        total="$(/bin/df -k -t ${types} | tail -n +1 | awk '{s+=$2} END {print s}')"
-        DISK_SIZE="$((total * 1024))"
+        DISK_SIZE=$(($(/bin/df -k -t ${types} | tail -n +1 | sed -r 's/\/dev\/disk([[:digit:]]*)s[[:digit:]]*/\/dev\/disk\1/g' | sort -u -k '1,2n,4n' | awk '{print $2}' | tr '\n' '+' | head -c -1) * 1024))
```

Comment:
Typo? There is no `-r` option in sed?
```suggestion
        DISK_SIZE=$(($(/bin/df -k -t ${types} | tail -n +1 | sed -e 's/\/dev\/disk([[:digit:]]*)s[[:digit:]]*/\/dev\/disk\1/g' | sort -u -k '1,2n,4n' | awk '{print $2}' | tr '\n' '+' | head -c -1) * 1024))
```

Reply:
```sh
$ ./daemon/system-info.sh
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
sort: stray character in field spec: invalid field specification ‘1,2n,4n’
./daemon/system-info.sh: line 308: * 1024: syntax error: operand expected (error token is "* 1024")
```

Reply:
Not technically a typo, it's just GNU sed specitic (which i forgot). Should be -E.